CHANGELOG.md

```diff
@@ -9,7 +9,7 @@
 * MapboxNavigation now depends on [MapboxMaps v10.0.0-beta.20](https://github.com/mapbox/mapbox-maps-ios/). ([#3010](https://github.com/mapbox/mapbox-navigation-ios/pull/3010))
 * MapboxNavigation now depends on MapboxNavigationNative v50.0.0. ([#3010](https://github.com/mapbox/mapbox-navigation-ios/pull/3010))
 * MapboxNavigation now depends on MapboxCommon v12.0.0. ([#3010](https://github.com/mapbox/mapbox-navigation-ios/pull/3010))
-* MapboxCoreNavigation depends on MapboxDirections v2.0.0-beta.2 but no longer depends on MapboxAccounts. ([#2808](https://github.com/mapbox/mapbox-navigation-ios/pull/2808), [#2829](https://github.com/mapbox/mapbox-navigation-ios/pull/2829), [#2837](https://github.com/mapbox/mapbox-navigation-ios/pull/2837))
+* MapboxCoreNavigation depends on MapboxDirections v2.0.0-beta.3 but no longer depends on MapboxAccounts. ([#2808](https://github.com/mapbox/mapbox-navigation-ios/pull/2808), [#2829](https://github.com/mapbox/mapbox-navigation-ios/pull/2829), [#2837](https://github.com/mapbox/mapbox-navigation-ios/pull/2837))
```

Comment:
Looks like this got rolled back for now.

Reply:
No, the new one is [beta.4](https://github.com/mapbox/mapbox-directions-swift/releases/tag/v2.0.0-beta.4). We're on [beta.3](https://github.com/mapbox/mapbox-navigation-ios/blob/release-v2.0/Package.swift#L24) now.